### ISSUE TITLE
 allObjects[i].set is not a function 	 fabric.js (строка 8207)

### ISSUE THREAD

**Issue by ramir1** [fabricjs/fabric.js#1066]
allObjects[i].set is not a function

fabric.js (строка 8207)

**Comment by desprit** [comment]
@kangax 

1. I added an overlay image to my canvas:
bodyCanvas.setOverlayImage(.....)
Now I place an image with this function:
fabric.util.loadImage(image, function(img) {})

2. Image is added on the canvas, can be resized, moved etc;

3. I do mouse down on the image, move it and when I do mouse up it doesn't release the image. I mean, I'm still like holding it even if I'm no longer clicking on my mouse. At the same time I see "allObjects[i].set is not a function" error in console.

p.s. This only happened to me when I changed my code to use overlay image. Previously I had a normal image added on the canvas with "evented" set to false and bringToTop() to emulate an overlay.
p.p.s. This only happens when I add an image to my canvas. I can add rects, circles, text, drag them, rotate etc and "allObjects[i].set is not a function" not happens.

Hope this will help somehow.
